test/fixtures/index.js

```diff
@@ -69,13 +76,15 @@ describe('custom resolver', () => {
           downName: 'down',
           migrations: {
             wrap: fn => () => fn(this.sequelize.getQueryInterface(), this.sequelize.constructor),
+            nameFormatter: (path) => parse(path).name,
```

Comment:
Question: Is this addition to this test necessary for it to pass? Would it fail without this?

Reply:
Yes, it would because then this check would fail as the file extensions are missing:
https://github.com/sequelize/umzug/blob/0cf1ee75dc6375647772d6ca5010b7fedacd2af2/test/fixtures/index.js#L56